docs/data/material/components/material-icons/SearchIcons.js

```diff
@@ -488,7 +488,7 @@ export default function SearchIcons() {
         if (value === '') {
           setKeys(null);
         } else {
-          searchIndex.searchAsync(value).then((results) => {
+          searchIndex.searchAsync(value, { limit: 3000 }).then((results) => {
```

Comment:
I have downgraded to version 0.6.32 and It works well, so it's definitely a regression from the upgrade from 0.6.x to 0.7.x.

Actually, in Flexsearch documentation you can see how now the default limit is now 100 if no parameter is added https://github.com/nextapps-de/flexsearch#search-options

I have added 3000 since the length of synonyms.js file is ~2897 lines, so it's fully covered.